node/network/protocol/src/lib.rs

```diff
@@ -438,6 +438,62 @@ pub mod v1 {
 	impl_try_from!(CollationProtocol, CollatorProtocol, CollatorProtocolMessage);
 }
 
+/// Unified annoyance cost and good behavior benefits.
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+#[allow(missing_docs)]
+pub enum UnifiedReputationChange {
+	CostMajor(&'static str),
+	CostMinor(&'static str),
+	CostMajorRepeated(&'static str),
+	CostMinorRepeated(&'static str),
+	Malicious(&'static str),
+	BenefitMinorFirst(&'static str),
+	BenefitMinor(&'static str),
+	BenefitMajorFirst(&'static str),
+	BenefitMajor(&'static str),
+}
+
+impl UnifiedReputationChange {
+	/// Obtain the cost or benefit associated with
+	/// the enum variant.
+	const fn cost_or_benefit(&self) -> i32 {
+		match self {
+			Self::CostMinor(_) => -10_000,
+			Self::CostMajor(_) => -30_000,
+			Self::CostMinorRepeated(_) => -20_000,
+			Self::CostMajorRepeated(_) => -60_000,
+			Self::Malicious(_) => -1_000_000,
+			Self::BenefitMajorFirst(_) => 30_000,
+			Self::BenefitMajor(_) => 20_000,
+			Self::BenefitMinorFirst(_) => 15_000,
+			Self::BenefitMinor(_) => 10_000,
```

Comment:
note that previously `BenefitMinor` was 10x smaller than -`CostMinor`, this was to account for one block having multiple messages leading to `BenefitMinor` reputation change, maybe should allow subsystem to divide these numbers freely

Reply:
Fixed